Formula/code-cli.rb

```diff
@@ -0,0 +1,23 @@
+class CodeCli < Formula
+  desc "Command-line interface built-in Visual Studio Code"
+  homepage "https://github.com/microsoft/vscode"
+  url "https://github.com/microsoft/vscode/archive/refs/tags/1.74.0.tar.gz"
+  sha256 "171435a9e2736d323af78591593f8a59eac52d26e98ba817daa37261e835aa1f"
+  license "MIT"
+  head "https://github.com/microsoft/vscode.git", branch: "main"
+
```

Comment:
need livecheck, otherwise it would report `code-cli: 1.74.0 ==> 20172701.01`
```suggestion
  livecheck do
    url :stable
    strategy :github_latest
  end

```